packages/main/src/components/AnalyticalTable/index.tsx

```diff
@@ -158,10 +159,14 @@ const AnalyticalTable: FC<TableProps> = forwardRef((props: TableProps, ref: Ref<
     minRows,
     isTreeTable,
     alternateRowColor,
-    overscanCount
+    overscanCount,
+    scaleWidthMode
   } = props;
 
   const classes = useStyles({ rowHeight: props.rowHeight });
+  const [internalRows, setInternalRows] = useState([]);
+  const [hiddenColumns, setHiddenColumns] = useState([]);
```

Comment:
Do we really need this shadow on the hidden columns? can't we just call `useDynamicColumnWidths` after `useTable` and use the columns from the tableState?